app/views/notifications/index.html.erb

```diff
@@ -19,55 +19,185 @@
+                <div class="crayons-dropdown__icon">
+                  <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+                </div>
+              </a>
+            </nav>
+          </div>
+
+        <% end %>
+      <% end %>
+
+      <div class="crayons-tabs notifications-mobile-filters <%= "showing" if params[:org_id].blank? %>">
+        <a class="crayons-tabs--item <%= "selected" if params[:filter].blank? %>" href="/notifications">
+          <span class="crayons-tabs--label">
+            All
+          </span>
+        </a>
+        <a class="crayons-tabs--item <%= "selected" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>" href="/notifications/comments">
+          <span class="crayons-tabs--label">
+            Comments
+          </span>
+        </a>
+        <a class="crayons-tabs--item <%= "selected" if params[:filter].to_s.casecmp("posts").zero? && params[:org_id].blank? %>" href="/notifications/posts">
+          <span class="crayons-tabs--label">
+            Posts
+          </span>
+        </a>
       </div>
+
       <% if @organizations.present? %>
         <% @organizations.each do |org| %>
-          <div class="notifications-mobile-filters">
-            <hr>
-            <span class="organization-name">@<%= org.slug %>:</span>
-            <a class="query-filter-button <%= "selected" if params[:filter].to_s.casecmp("org").zero? && params[:org_id].to_i == org.id %>" href="/notifications/org/<%= org.id %>">ALL</a>
-            <a class="query-filter-button <%= "selected" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].to_i == org.id %>" href="/notifications/comments/<%= org.id %>">COMMENTS</a>
+          <div class="crayons-tabs notifications-mobile-filters <%= "showing" if params[:org_id].to_i == org.id %>">
+            <a class="crayons-tabs--item <%= "selected" if params[:filter].to_s.casecmp("org").zero? && params[:org_id].to_i == org.id %>" href="/notifications/org/<%= org.id %>">
+              <span class="crayons-tabs--label">
+                All
+              </span>
+            </a>
+            <a class="crayons-tabs--item <%= "selected" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].to_i == org.id %>" href="/notifications/comments/<%= org.id %>">
+              <span class="crayons-tabs--label">
+                Comments
+              </span>
+            </a>
           </div>
         <% end %>
       <% end %>
 
+      </section>
       <%= render "notifications_list", params: params %>
+
     </div>
 
     <%# new or less active users that don't have a page worth of notifications
         won't be shown the "load more" button %>
     <% if @notifications.size >= @initial_page_size %>
       <div class="load-more-wrapper">
-        <button id="load-more-button" type="button">
+        <button id="load-more-button" type="button" class="crayons-btn--secondary">
```

Comment:
I think you need two classes here: `crayons-btn crayons-btn--secondary` (BEM naming).

Reply:
OoOooOoo okay!